src/components/GenericWarningModal.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+import LoadingButton from './LoadingButton';
+
+
+const GenericWarningModal = ({ displayTitle, displayText, onProceed, onCancel, type }) => (
+  <div className={elementStyles.overlay}>
+    <div className={elementStyles['modal-warning']}>
+      <div className={elementStyles.modalHeader}>
+        <h1>
+          {displayTitle}
+          {' '}
+          { type }
```

Comment:
What is type used for? Might be good to clarify

Reply:
Good catch, removed `type` in [4dea0ad](https://github.com/isomerpages/isomercms-frontend/pull/170/commits/4dea0ad27123417ebc7dce3c1c822771f2cc1817)